pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -81,6 +81,8 @@ When using third party authentication systems, Code.org may receive personal dat
 
 Under a student's control, they can dissociate or associate themselves with as many teachers as they want, (and those teachers get access to the student's course progress and display name but not their email address). Teachers who create User accounts on behalf of students can reset their “secret word” or “secret picture” (for young children who can’t read) for as long as the student leaves the teacher in control of the account.
 
+Our collection, use, and disclosure of Student Data is governed by our Privacy Policy and Terms of Use by the provision of the Family Education Rights and Privacy Act (FERPA), the Children's Online Privacy Protection Act (COPPA), and applicable state laws which relate to the collection of Student Data including the Student Online Personal Information Protection Act (SOPIPA). 
```

Comment:
Should it be Family Educational instead of Family Education (for FERPA)?

Reply:
Good catch! Thank you!! Fixed.